### PR TITLE
chore: upgrade dependencies

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**

### CODE REVIEW

**Review by alantoa — APPROVED** [review]
LGTM! ✨

File: apps/expo/package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
     "domain-browser": "^4.22.0",
     "dotenv": "^10.0.0",
     "events": "^3.3.0",
-    "expo": "next",
+    "expo": "~46.0.7",
```
Comment (alantoa) [inline apps/expo/package.json:135]: yeah, patch files will have some issues when the lib version has changed if we use `next`. 

for example: `react-native+0.69.3.patch` file, the expo `next` version is used react-native `v0.69.4`, If we forget to update this file, this patch will not be used.

Comment (alantoa) [reply apps/expo/package.json:135]: so I'm not sure if `expo` `~46.0.7` uses` react-native v0.69.4`.

Comment (intergalacticspacehighway) [reply apps/expo/package.json:135]: good catch, will update the patch!

> so I'm not sure if expo ~46.0.7 uses react-native v0.69.4.

yeah, it's uses 0.69.4

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Pull request merged**